frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -81,6 +81,7 @@
           </div>
           <div
             v-if="
+              nominationStatus &&
               (nominationStatus.tried || nominationStatus.solved) &&
```

Comment:
```suggestion
              (nominationStatus?.tried || nominationStatus?.solved) &&
```

esto funciona? si sí, hay que usarlo porque es más conciso!

Reply:
Parece que sí.